[IMP] POS: Update Restaurant features (#14690)
odoo/documentation: content/applications/sales/point_of_sale/restaurant.rst

Order transfer
--------------

To transfer an order to another table from the :ref:`POS register <pos/restaurant/orders>`, click
the :icon:`fa-ellipsis-v` (:guilabel:`Actions`) icon, then :guilabel:`Transfer/Merge`, and choose
the target table in the :ref:`Floor plan <pos/restaurant/floors>` view:

   - Select an available table to transfer customers and their orders.
   - Select an occupied table to merge customers and their orders.

.. _pos/restaurant/orders/preset:

Presets
-------

Presets are preconfigured settings that determine whether an order is for :guilabel:`Dine In`,
:guilabel:`Takeout`, and :guilabel:`Delivery`. They also control whether customer contact
information is required and apply capacity limits based on opening hours and order quantity.

.. seealso::
   - :doc:`/applications/sales/point_of_sale/preparation`
   - :doc:`/applications/sales/point_of_sale/online_food_delivery`

.. _pos/restaurant/orders/preset/backend:

Preset editing
~~~~~~~~~~~~~~

To edit preconfigured presets, follow these steps:

> #. Go to the :ref:`POS settings <configuration/settings>` and scroll down to the the
>    :guilabel:`Take out / Delivery / Members` setting under the :guilabel:`Point of Sale` section.

Reviewer: ```suggestion
#. Go to the :ref:`POS settings <configuration/settings>` and enable to the the
   :guilabel:`Take out / Delivery / Members` setting under the :guilabel:`Point of Sale` section.
```
And since you can't leave the settings (i.e., click configure presets) without saving, and you can't save unless you have specified a Default preset (...), I would add the Available and Default fields as a separate step right under this one? (and don't forget to save)

#. Click :icon:`oi-arrow-right` :guilabel:`Configure Presets` under the :guilabel:`Default` field,

>    then click the desired preset.

Reviewer: ```suggestion
   then select the desired preset or click :guilabel:`New`.
```

- :guilabel:`Dine In`: Set a :ref:`table or a tab <pos/restaurant/orders/process>`.

- :guilabel:`Takeout`: Enter the customer's name and click :guilabel:`Apply`, then select a date
  and a time slot.

- :guilabel:`Delivery`: Choose a customer. To create one, click :guilabel:`Create`, enter a name,
  phone number, an email, and an address, then click :guilabel:`Save`.

.. _pos/restaurant/orders/courses:

Courses
-------

Odoo Point of Sale allows orders to be split into multiple courses, sending each course to the
kitchen in sequence when using the :guilabel:`Dine In` :ref:`preset
<pos/restaurant/orders/preset/use>`.

To split an order into courses from the :ref:`register <pos/restaurant/orders>`, click
:guilabel:`Course` and start adding products. Repeat the action as many times as needed. Set a
:ref:`table or a tab <pos/restaurant/orders/process>`, then click :guilabel:`Order` to send the
order to the kitchen, which also fires the first course.

When ready for the second course, retrieve the order from the :ref:`Floor plan
<pos/restaurant/floors>` view or the :ref:`Orders overview <pos/restaurant/orders-overview>`, and

> click :guilabel:`Fire Course 2`. Repeat the action as many times as needed.

Reviewer: - I would say *add products* instead of *start adding products*
- I don't see an *Order* button, just *Fire Course 1/2/3/etc* (depending on which course you select in the cart)
- I would remove the part about set table or set tab as this is part of the "basic" order process and already mentionned in the dedicated section on this page (also, people might have already done this before adding courses anyways so the buttons might not even be there at this point)
- You can also click Course multiple times to add the courses, then click a course in the cart and add products tot hat course, then click Course 2 and add products, etc. You can then just click Fire Course 1 to send that first course to the kitchen. 

I would review this paragraph based on these comments

.. tip::
   To transfer a product or an entire course into another course, select it in the cart, click the
   :icon:`fa-ellipsis-v` (:guilabel:`Actions`) icon, then :icon:`fa-arrow-down`
   :guilabel:`Transfer course`, and select the preferred course.

.. _pos/restaurant/orders-overview:

Orders overview
===============

> The :guilabel:`Orders` overview allows for viewing, searching, retrieving, and loading all orders.

Reviewer: ```suggestion
The :guilabel:`Orders` overview allows for viewing, searching, and retrieving orders.
```
retrieving and loading mean the same thing here imo

To view orders specific to a preset, click the relevant preset.

To search for orders, type the :guilabel:`Reference`, :guilabel:`Receipt Number`, :guilabel:`Date`,
or :guilabel:`Customer` in the search bar.

To filter orders based on their status, click the dropdown menu and select one of the following
options:

- :guilabel:`Active`: Orders in progress, either marked as :guilabel:`Ongoing` or in the
  :guilabel:`Payment` stage.
- :guilabel:`Receipt`: Orders with the receipt emailed to the customer.
- :guilabel:`Paid`: Paid orders.

> To navigate between pages, click the :icon:`fa-caret-left` :icon:`fa-caret-right`
> (:guilabel:`caret`) icons.

Reviewer: ```suggestion
To navigate between pages, click the :icon:`fa-caret-left` or :icon:`fa-caret-right` (:guilabel:`caret`) icon.
```

To access an order in the register, click it, then click :guilabel:`Load Order`.

.. note::
   Paid orders can be :ref:`refunded <pos/refund>`.

.. tip::
   - To define the number of orders visible on a page, click `1-x / x`. Enter a number lower than
     the total number of pages, and click :guilabel:`Ok`.
   - Click the :icon:`fa-trash` (:guilabel:`trash`) icon next to an :guilabel:`Active` order to
     delete it.

>    - Click the preset again to go back to the main overview.

Reviewer: what if people don't use presets?

     image, determine the product availability, then click :guilabel:`Save & Close`.

.. tip::
   - To access all preparation printers from the :ref:`POS settings <configuration/settings>`,
     scroll down to the :guilabel:`Preparation` section and click :icon:`oi-arrow-right`
     :guilabel:`Printers`. Alternatively, go to :menuselection:`Point of Sale --> Orders -->
     Preparations Printers`.
   - In case of a printing issue, click the :icon:`fa-cutlery` (:guilabel:`order`) icon in the

>      :ref:`POS register <pos/restaurant/orders>` to reprint the last kitchen order.

Reviewer: where exactly?
